src/YieldDisburser.sol

```diff
@@ -10,60 +10,115 @@ import {Bread} from "../lib/bread-token-v2/src/Bread.sol";
 error AlreadyClaimed();
 
 contract YieldDisburser is OwnableUpgradeable {
-    address[] public breadchainProjects;
-    address[] public queuedProjectsForAddition;
-    address[] public queuedProjectsForRemoval;
-    address[] public breadchainVoters;
+    // Storage of intristic , assumed constants
+    // @notice The address of the Bread token contract
     Bread public breadToken;
-    uint48 public lastClaimedTimestamp;
-    uint256 public lastClaimedBlocknumber;
+    // @notice The block time of the evm in seconds
+    uint256 public blockTime;
```

Comment:
move this below "Storage of configuration variables"
fwiw, this is generally why i avoid categorizing things underneath comments like this. it sort of creates more ambiguity than it solves because variables tend to accidentally get caught underneath descriptive comments that never get "terminated"